### PR TITLE
Validate the task is running before streaming the logs

### PR DESCRIPTION
**Pull request by IvanPombo**
This PR validates the task is running before starting the logs stream.

# QA

Try to check a task that is not yours: `inductiva logs pineapple`.
Try to check a task that has finished: `inductiva logs <task_id_of_terminal_task>`.
Try to check a task that is running: `inductiva logs <task_id_running>`

### CODE REVIEW

**Review by Pbarbosa92 — APPROVED** [review]

File: inductiva/tasks/streams.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 from inductiva import constants
 
 logger = logging.getLogger("websocket")
-logger.setLevel(logging.INFO)
+logger.setLevel(logging.DEBUG)
```
Comment (AugustoPeres) [inline inductiva/tasks/streams.py:13]: Why the change here?

**Review by AugustoPeres — APPROVED** [review]
LGTM. WIll QA now

File: inductiva/tasks/streams.py (hunk 1 above)
Comment (IvanPombo) [reply inductiva/tasks/streams.py:13]: The reason is to reduce the ping/pong messages on reconnect, but Sérgio will address this soon.

**Comment by Pbarbosa92** [comment]
**QA**

Finished task:
Captura de ecrã 2024-02-01, às 17 08 01
Checking a task that is not mine (is this the expected result ? or is this kind of error addressed in another issue ?):
Captura de ecrã 2024-02-01, às 17 09 01
Check running task:
Captura de ecrã 2024-02-01, às 17 10 32

**Comment by AugustoPeres** [comment]
### QA ✅ 

My task
Screenshot 2024-02-01 at 17 17 14

Finished task
Screenshot 2024-02-01 at 17 17 54

Unauthorised task
Screenshot 2024-02-01 at 17 18 22

File: inductiva/tasks/streams.py [hunk 2]
```diff
@@ -82,6 +82,13 @@ def __on_open(self, unused_ws):
...
+        if inductiva.tasks.Task(self.task_id).is_running():
+            self.ws.run_forever(reconnect=self.RECONNECT_DELAY_SEC,
+                                ping_interval=self.PING_INTERVAL_SEC,
+                                ping_timeout=self.PING_TIMEOUT_SEC)
+        else:
+            raise RuntimeError(
+                f"Task {self.task_id} has terminated running and the simulation"
+                " logs are no longer available for streaming. \n Check the "
+                "task status with \n\t inductiva tasks list --task-id "
+                f"{self.task_id}")
```
Comment (sergiomsantos) [inline inductiva/tasks/streams.py:94]: This check should have gone into the `inductiva logs` command.
